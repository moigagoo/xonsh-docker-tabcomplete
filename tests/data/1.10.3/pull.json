{
        "arguments": [
                "-a", "--all-tags",
                "--disable-content-trust=true",
                "--help"
        ],
        "commands": []
}
