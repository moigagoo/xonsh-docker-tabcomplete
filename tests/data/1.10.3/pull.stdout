
Usage:	docker pull [OPTIONS] NAME[:TAG|@DIGEST]

Pull an image or a repository from a registry

  -a, --all-tags                  Download all tagged images in the repository
  --disable-content-trust=true    Skip image verification
  --help                          Print usage
